### ISSUE TITLE
Training recipe for these weights

### ISSUE THREAD

**Issue by briancheung** [pytorch/vision#5945]
https://github.com/pytorch/vision/blob/62740807c18e68bb0acd85895dca527f9a655bd5/torchvision/models/vision_transformer.py#L377

Does anyone know how these weights were generated. Where they training from scratch only on ImageNet 1k or was it pre-trained on ImageNet 21k? Looking at the original Vision transformer paper: https://arxiv.org/abs/2010.11929 I'm not quite sure where the accuracy numbers in these lines are coming from:

```python
class ViT_B_32_Weights(WeightsEnum):
    IMAGENET1K_V1 = Weights(
        url="https://download.pytorch.org/models/vit_b_32-d86f8d99.pth",
        transforms=partial(ImageClassification, crop_size=224),
        meta={
            **_COMMON_META,
            "num_params": 88224232,
            "min_size": (224, 224),
            "recipe": "https://github.com/pytorch/vision/tree/main/references/classification#vit_b_32",
            "metrics": {
                "acc@1": 75.912,
                "acc@5": 92.466,
            },
        },
    )
    DEFAULT = IMAGENET1K_V1
```

Here's the corresponding numbers presented in the original Vision Transformer paper, ViT-B/32 accuracy of 75.912 is not in either the ImageNet 1k or the ImageNet 21k columns:

![image](https://user-images.githubusercontent.com/1216594/166825518-d0279f89-b604-4ec7-9b61-ca13da794a71.png)

cc @datumbox

**Comment by briancheung** [comment]
Thank you, that clarifies things!
